### PR TITLE
BAH-1773 | Lab Tests

### PR DESCRIPTION
**Pull request by sivareddyp**
JIRA - https://bahmni.atlassian.net/browse/BAH-1773
List of tests - https://docs.google.com/spreadsheets/d/1S42ei4PcbaSwWG37k1b6G3V2lM9X_AzBXzegTxk_8SQ/edit?pli=1#gid=1444056136

### CODE REVIEW

File: masterdata/configuration/liquibase/liquibase.xml [hunk 1]
```diff
@@ -306,4 +306,21 @@
             "org.openmrs.customdatatype.datatype.FreeTextDatatype", 0, 1, NOW(), 0, "07421454-e196-422d-a7c3-a045ffc5fac8");
         </sql>
     </changeSet>
+    <changeSet id="BAH-1773-labtests-20221107" author="Clinic-Config">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from order_type_class_map join order_type on order_type_class_map.order_type_id = order_type.order_type_id
```
Comment (angshu) [inline masterdata/configuration/liquibase/liquibase.xml]: please dont do count(*) especially in a join case. Use a specific column

File: masterdata/configuration/liquibase/liquibase.xml [hunk 2]
```diff
@@ -306,4 +306,21 @@
...
+                join concept_class cc on order_type_class_map.concept_class_id = cc.concept_class_id where cc.name='Test';
```
Comment (angshu) [inline masterdata/configuration/liquibase/liquibase.xml]: Why are we using "Test" class with orders. We typically use "LabTest" (single test) and "LabSet" (panel). If you change this, then lots of places we may have to change the code

Comment (sivareddyp) [reply masterdata/configuration/liquibase/liquibase.xml]: @angshu All these concepts are from CIEL dictionary and individual test having class as **Test**. So, added class **Test** to _order_type_class_map_ class. 
Reference Document - https://bahmni.atlassian.net/wiki/spaces/BAH/pages/32014568/Configure+Orders+Tab+for+Lab+and+Radiology

File: masterdata/configuration/liquibase/liquibase.xml (hunk 1 above)
Comment (sivareddyp) [reply masterdata/configuration/liquibase/liquibase.xml]: updated @angshu .

File: masterdata/configuration/liquibase/liquibase.xml (hunk 2 above)
Comment (gsluthra) [reply masterdata/configuration/liquibase/liquibase.xml]: So @sivareddytw - you are saying that if this mapping to "Test" class is done, then Bahmni Order Test scenarios, etc are working fine? Including Sync to OpenELIS and back?

Comment (sivareddyp) [reply masterdata/configuration/liquibase/liquibase.xml]: @gsluthra @angshu we have tested this and found that lab lite is not functioning properly. Lab Lite expecting LabTest/LabSet classes. we need to discuss on the next steps. 
cc: @gokultw

Comment (sivareddyp) [reply masterdata/configuration/liquibase/liquibase.xml]: @gsluthra @angshu  removed liquibase migration as we are updating concept class from **Test** to **LabTest** in the PR https://github.com/BahmniIndiaDistro/clinic-config/pull/131

**Review by gsluthra — APPROVED** [review]

**Pull request merged**